pages/deployment/deployment.md

```diff
@@ -42,7 +42,7 @@ specified, it defaults to running `node server.js`.)
 A good example of this is the [text-to-speech][text-to-speech] application -
 here's its `package.json` file*:
 
-```json
+```JSON
```

Comment:
If it works for you locally as expected, should be good to merge

On Tue, Aug 16, 2016 at 7:30 PM, Shaun Mulligan <notifications@github.com>
wrote:

> In pages/deployment/deployment.md
> <https://github.com/resin-io/docs/pull/294#discussion_r74970287>:
>
> > @@ -42,7 +42,7 @@ specified, it defaults to running `node server.js`.)
> >  A good example of this is the [text-to-speech][text-to-speech] application -
> >  here's its `package.json` file*:
> >
> > -```json
> > +```JSON
>
> @emirotin <https://github.com/emirotin> not really sure, i just saw it in
> uppercase in the listing on the highlight.js
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/resin-io/docs/pull/294/files/2acc6135b887d1ca456c7080999b30708de78ac8#r74970287>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgGCNZuY5vLgJQS0pgzvg9-xZvLtEc2ks5qgeWagaJpZM4JlUM6>
> .
>



-- 
Eugene Mirotin
Senior Frontend Engineer
site: Resin.io <https://resin.io/>, twitter: @resin_io
<https://twitter.com/resin_io>